app/assets/javascripts/gemstore-app/controllers/store_ctrl.js

```diff
@@ -0,0 +1,25 @@
+(function(){
+  var app = angular.module('gemStoreApp');
+  app.controller('StoreController', [ '$scope', '$http', function($scope, $http) {
+    $scope.products = [];
+    $scope.errors = [];
+
+    $scope.index = function() {
+      $http.get( "/api/v1/gemstones" )
+        .success(function(data) {
+          $scope.products = data;
+        });
+    };
+
+    $scope.create = function(gemstone){
+      $http.post( "/api/v1/gemstones", { gemstone: gemstone })
+        .success(function(data) {
+          $scope.products.push(data);
+        });
+        // .error(function(data) {
+        //   $scope.errors.push(something);
+        // })
```

Comment:
If this is something to come back to, a conventional way of notating is with a comment like... `//TODO: add .error callback`. 

That lets you grep for "TODO" in your app to find things left undone. A comment like this is more ambiguous.